app.ts

```diff
@@ -291,7 +291,7 @@ async function setupWebPackDevMiddleware(router) {
     const webpackCompiler = webpack(webpackConfig);
     router.use(webpackDevMiddleware(webpackCompiler, {
         publicPath: '/static',
-        logger: logger,
+        //logger: logger,
```

Comment:
Did we lose the logger here? Temporarily?

Reply:
Weird, I don't remember doing this...